### PR TITLE
Add project descriptions with links, change background, fav facts, fixed some styling

### PR DESCRIPTION
**Pull request by ruchiishahh**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -26,3 +26,57 @@ function addRandomGreeting() {
...
+        this.el = el;
+        this.loopNum = 0;
+        this.period = parseInt(period, 10) || 2000;
+        this.txt = '';
+        this.tick();
+        this.isDeleting = false;
+    };
+
+    TxtType.prototype.tick = function() {
+        var i = this.loopNum % this.toRotate.length;
```
Comment (bozhangbz) [inline portfolio/src/main/webapp/script.js:42]: https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -26,3 +26,57 @@ function addRandomGreeting() {
...
+        var fullTxt = this.toRotate[i];
+
+        if (this.isDeleting) {
+        this.txt = fullTxt.substring(0, this.txt.length - 1);
```
Comment (bozhangbz) [inline portfolio/src/main/webapp/script.js:46]: https://google.github.io/styleguide/jsguide.html#formatting-block-indentation

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -26,3 +26,57 @@ function addRandomGreeting() {
...
+        this.toRotate = toRotate;
+        this.el = el;
+        this.loopNum = 0;
+        this.period = parseInt(period, 10) || 2000;
+        this.txt = '';
+        this.tick();
+        this.isDeleting = false;
+    };
+
+    TxtType.prototype.tick = function() {
```
Comment (bozhangbz) [inline portfolio/src/main/webapp/script.js:41]: https://google.github.io/styleguide/jsguide.html#features-classes-prototypes

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -26,3 +26,57 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+
+var TxtType = function(el, toRotate, period) {
```
Comment (bozhangbz) [inline portfolio/src/main/webapp/script.js:31]: https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -10,42 +17,172 @@
...
 body{
-    margin: auto;
+    align-items: center;
+    background-color: #fac564;
     background-color: lightblue;
+    background-image: url(https://s3-us-west-2.amazonaws.com/s.cdpn.io/5908/food.png);
+    display: flex;
+    font-family: "Montserrat", sans-serif;
+    flex-direction: column;
+    font-smoothing: antialiased;
```
Comment (laskdaf) [inline portfolio/src/main/webapp/style.css:27]: alphabetical order, also check the rest

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -10,42 +17,172 @@
...
 
-h1{
+p.desc{
+    border-style: double;
+    border-width: thin;
+    border-color: green;
     text-align: center;
+}
+
+
```
Comment (laskdaf) [inline portfolio/src/main/webapp/style.css:40]: extra space
```suggestion
```

File: portfolio/src/main/webapp/style.css [hunk 7]
```diff
@@ -10,42 +17,172 @@
...
+*, *:before, *:after{
+  box-sizing:border-box;
+}
+
+div.master-title{
+  width: 100%;
+  margin: 0 auto;
+  text-align: center;
+
+}
```
Comment (laskdaf) [inline portfolio/src/main/webapp/style.css:57]:
```suggestion
}

```

File: portfolio/src/main/webapp/style.css [hunk 8]
```diff
@@ -10,42 +17,172 @@
...
 
+*, *:before, *:after{
+  box-sizing:border-box;
+}
+
+div.master-title{
+  width: 100%;
+  margin: 0 auto;
+  text-align: center;
+
```
Comment (laskdaf) [inline portfolio/src/main/webapp/style.css:56]:
```suggestion
```

File: portfolio/src/main/webapp/index.html [hunk 9]
```diff
@@ -2,22 +2,49 @@
...
+        <span class="letter" data-letter="I">I</span>
+        <span class="letter" data-letter="S">S</span>
+        <span class="letter" data-letter="H">H</span>
+        <span class="letter" data-letter="A">A</span>
+        <span class="letter" data-letter="H">H</span>
+      </div>
+
+      <!-- <p class="portfolio_text">This is my personal portfolio. Scroll to learn more about me.</p> -->
+      <h2>
+        <center> <p class="typewrite" data-period="2000" data-type='[ "Hi, Nice to meet you.", "Welcome to my Webpage.", "Excited to show you my portfolio.", "Enjoy!" ]'>
```
Comment (laskdaf) [inline portfolio/src/main/webapp/index.html:27]: consider not using the `<center>` tags. it's old tech that's not supported in html5.
consider using flexbox: https://css-tricks.com/snippets/css/a-guide-to-flexbox/

**Pull request merged**